client/js/components/month/month.js

```diff
@@ -1,12 +1,34 @@
 /** @jsx React.DOM */
 var React = require('react');
-var MonthCalendarContainer = require('./month-calendarcontainer.js');
+var MonthEventsContainer = require('./month-eventcontainer.js');
```

Comment:
The CalendarContainer was redundent so has been removed entirely, eventscontainer is taking its place